doc/source/default-settings.rst

```diff
@@ -414,6 +414,12 @@ Bibtex options
    check will show an error if the key does not have the corresponding type. The
    type should be a builtin Python type that can be used with ``eval``.
 
+.. papis-config:: doctor-key-type-check-separator
+
+   A separator used by the ``key-type`` check when the ``type`` is :class:`list`
+   to split or join strings. The split will ignore additional whitespace around the
+   separator.
+
```

Comment:
Maybe this could benefit from a little more explanation, à la: A separator used by the ``key-type`` check. When converting from :class:`str` to :class:`list`, it is used to split the string into list items, and when converting from :class:`list` to :class:`str`, it is used to join list items to form a string. The split will ignore additional whitespace around the separator (for instance, when set to `` , ``, the string ``"extra,    whitespace"`` will be converted to the list ``["extra", "whitespace"]``). To preserve leading and trailing whitespace in the separator, make sure to quote it (for instance, ``", "``).

Reply:
Not sure about the `:class:` syntax, so change that if I used that wrongly...

Reply:
It was all correct -- just copied this verbatim :grin: Thanks!